wasm-host/src/host_functions_wamr.rs

```diff
@@ -301,8 +301,8 @@ pub fn account_keylet(
         return HostError::InvalidAccount as i32;
     }
     let keylet_hash = index_hash(LedgerNameSpace::Account, &data);
-    // let hex_str = hex::encode(&keylet_hash);
-    // println!("Data (keylet_hash): {:?}", hex_str);
+    let hex_str = hex::encode(&keylet_hash);
+    println!("Data (keylet_hash): {:?}", hex_str);
```

Comment:
The above 2 lines should stay commented out, since all other projects and examples will see those prints too.